visualization_scripts/activate.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env/bash
+
+#conda init bash
+echo "starting script"
+eval "$(conda shell.bash hook)"
+conda activate py2cyto
+
+echo "check enviornment"
```

Comment:
You may need to remove this `echo` call.  Currently the test for the environment doesn't work.  `$?` will refer to the status from the `echo` call instead of the `conda activate` call.  See https://unix.stackexchange.com/questions/7704/what-is-the-meaning-of-in-a-shell-script

Reply:
good catch, I removed this from the script and the environment can be created successfully.